observability/opentelemetry-metrics-backend/src/main/scala/sttp/client4/opentelemetry/OpenTelemetryMetricsBackend.scala

```diff
@@ -195,6 +211,32 @@ private class OpenTelemetryMetricsListener(
     b = config.description.fold(b)(b.setDescription)
     b.build()
   }
+
+  private def createRequestAttributes(request: GenericRequest[_, _]): Attributes = {
+    val attributes = Attributes
```

Comment:
maybe you could add links to appropriate section of the OT specification, just for reference why we're setting these specific attributes?

Reply:
(as comments)

Reply:
or simply a single link to the page listing the attributes